BUILDING.md

```diff
@@ -0,0 +1,249 @@
+| audacity_use_pa_jack              | STRING | linked     | Use the JACK audio interface if available [loaded, linked, off] |
+| audacity_use_pa_oss               | BOOL   | YES        | Use the OSS audio interface if available                        |
+| audacity_use_pch                  | BOOL   | YES        | Use precompiled headers [yes, no]                               |
+| audacity_use_portaudio            | STRING | local      | Use portaudio library [local]                                   |
+| audacity_use_portmixer            | STRING | local      | Use portmixer library [local, off]                              |
+| audacity_use_portsmf              | STRING | local      | Use portsmf library [system (if available), local, off]         |
+| audacity_use_sbsms                | STRING | local      | Use sbsms library [system (if available), local, off]           |
+| audacity_use_sndfile              | STRING | local      | Use sndfile library [system (if available), local]              |
+| audacity_use_soundtouch           | STRING | local      | Use soundtouch library [system (if available), local, off]      |
+| audacity_use_soxr                 | STRING | local      | Use soxr library [system (if available), local]                 |
+| audacity_use_sqlite               | STRING | local      | Use sqlite library [system (if available), local]               |
+| audacity_use_twolame              | STRING | local      | Use twolame library [system (if available), local, off]         |
+| audacity_use_vamp                 | STRING | local      | Use vamp library [system (if available), local, off]            |
+| audacity_use_vorbis               | STRING | local      | Use vorbis library [system (if available), local, off]          |
+| audacity_use_vst                  | BOOL   | ON         | Use VST2 plug-in support [on, off]                              |
+| audacity_use_wxwidgets            | STRING | local      | Use wxwidgets library [system (if available), local, off]       |
+| audacity_use_zlib                 | STRING | system     | Use zlib library [system (if available), local, off]            |
+
+
+### Building using system libraries
+
+On Linux, it is possible to build Audacity using (almost) only the libraries provided by the package manager. We require the following version of the packages to be installed:
+
+| Name              | Version       | Ubuntu 20.04 package                         |
+| :---------------- | :------------ | :------------------------------------------- |
+| wxWidgets         | >= 3.1.3      | **N/A**                                      |
+| expat             | >= 2.2.9      | libexpat1-dev                                |
+| libmp3lame        | >= 3.100      | libmp3lame-dev                               |
+| libsndfile        | >= 1.0.28     | libsndfile-dev                               |
```

Comment:
This is more information that would be better in a spreadsheet.

Ideally the CMake configuration would simply fail with an appropriate error message if the correct library version is not available:

```
Error: expat version 2.2.9 or later is required. Please install it with this command:

  $ sudo apt install libexpat1-dev
```

You can give the command or link to the Wiki page that has the command. This [PR for MuseScore](https://github.com/musescore/MuseScore/pull/6262) does something similar if Qt is not installed.

Reply:
>This is more information that would be better in a spreadsheet.

We have a spreadsheet, but I guess we need an "offline" version as well. I will move this table into a separate doc under the `linux` directory.